src/notion/blocks.ts

```diff
@@ -40,13 +40,13 @@ export function paragraph(text: RichText[]): Block {
   } as Block;
 }
 
-export function code(text: RichText[]): Block {
+export function code(text: RichText[], lang = 'javascript'): Block {
```

Comment:
It would be more accurate, but it looks like the Notion API requires you to set the language. What would be a more appropriate fallback language?

https://developers.notion.com/reference/block#code-blocks

Reply:
Woops, you're right! Nevermind, this is the best way to do it then 👍🏻